src/components/BuiltWithXmtp/ListOfDevs.json

```diff
@@ -0,0 +1,76 @@
+ [
+   {
+      "name":"XMTP.chat",
+      "href":"https://xmtp.chat",
+      "image":"/img/builtWithXmtp/xmtp.svg",
+      "bgColor": "",
+      "description":"XMTP.chat is an example chat application demonstrating the core capabilities of the XMTP client SDK.",
+      "github": "https://github.com/xmtp/example-chat-react",
+      "twitter": "https://twitter.com/xmtp_"
+   },
+   {
+      "name":"Lenster",
+      "href":"https://lenster.xyz",
+      "image":"/img/builtWithXmtp/lenster.svg",
+      "bgColor": "bg-[#8B5CF6]",
+      "description":"Lenster is a composable, decentralized, and permissionless social media webapp built with Lens Protocol.",
+      "github": "https://github.com/lensterxyz/lenster",
+      "twitter": "https://twitter.com/lensterxyz"
+   },
+   {
+      "name":"Beep3r",
+      "href":"https://bb3.xyz/",
+      "image":"/img/builtWithXmtp/beeper.png",
+      "description":"Bringing back the joy of owning a 90s pager device and the original vision of twttr - text networks.",
+      "github": "https://github.com/FUTUREPRIMITIVEXYZ/bb3-beep3r",
+      "twitter": "https://twitter.com/bb3labs"
+   },
+   {
+      "name":"meTokens",
+      "href":"https://metokens.com/discover",
+      "image":"/img/builtWithXmtp/meTokens2.svg",
+      "description":"meTokens is a protocol built on Ethereum that allows creators and builders to issue their own personal token.",
+      "github": "https://github.com/meTokens/meTokens-core",
+      "twitter": "https://twitter.com/meTokens"
+   },
+   {
+      "name":"Relay Receiver",
+      "href":"https://relay.cc/",
+      "image":"/img/builtWithXmtp/relay.png",
+      "description":"Relay’s goal is to provide open-source tools and products on top of XMTP, and create the best place for web3 identities.",
+      "github": "https://github.com/relaycc",
+      "twitter": "https://twitter.com/relay_eth"
+   },
+   {
+      "name":"Lensport",
+      "href":"https://lensport.io/",
+      "image":"/img/builtWithXmtp/lensport.png",
+      "description":"Discover, collect, and sell Lens Posts to invest in creators you support. Built on Lens Protocol.",
```

Comment:
Do we want to add a GitHub URL here - or are we intentionally leaving it out for now? =)

Reply:
leaving it out some projects dont have a public repo available